meson.build

```diff
@@ -88,9 +88,8 @@ libwpe = library('wpe-' + api_version,
 	dependencies: dependencies,
 	version: soversion,
 	soversion: soversion_major,
-	include_directories: 'include',
```

Comment:
Why do we remove this?

Reply:
Because we don't need it given that this patch changes the `.c` sources to use relative paths to the actual location of the header files, for example `#include "../include/wpe/input.h"` (this is related to the other change you commented on below).